src/components/Countries/Countries.js

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
- [ OPTIONAL ] You can add a loading state while waiting for the data to load from the API. You can also add a conditional rendering to show a loading spinner or a message while the data is being fetched.